fetch/src/1_fetch.r

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Can you add the number to the `fetch` file directory, so that the step order is clear in the naming conventions, and use a more specific name for the script that indicates what the function(s) in that script do? In the end we'd like to have a convention where the script is saved as something like `1_fetch/src/download_sb_data.R`